packages/nimble-components/src/banner/tests/banner.stories.ts

```diff
@@ -125,12 +123,6 @@ export const _banner: StoryObj<BannerArgs> = {
             name: 'title-hidden',
             description: 'If set, hides the provided title.'
         },
```

Comment:
For the number field story we kept a parameter called "localized labels" to link the client to the relevant label tokens for that component. Could we do the same for the banner?

Reply:
The banner Storybook page already has the `Localizable labels` section added from a previous PR.